<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-UB3S2NO" failures="1" tests="2" name="Negative Login ch" time="14.932" errors="0" timestamp="27 May 2019 20:43:17 GMT">
  <testcase classname="com.portal.test.Testing" name="negativeLoginTest" time="1.131"/>
  <testcase classname="com.portal.test.Testing" name="negativeLoginTest" time="4.762">
    <failure type="java.lang.AssertionError" message="Error Message isn&amp;apos;t expected.
 Expected: Invalid User/Password
Actual:You are already logged in expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error Message isn't expected.
 Expected: Invalid User/Password
Actual:You are already logged in expected [true] but found [false]
at com.portal.test.Testing.negativeLoginTest(Testing.java:93)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- negativeLoginTest -->
  <testcase classname="com.portal.test.Testing" name="positiveLoginTest" time="0.0">
    <ignored/>
  </testcase> <!-- positiveLoginTest -->
</testsuite> <!-- Negative Login ch -->
